.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   plugins: ['@typescript-eslint', 'detox'],
   ignorePatterns: [
     '**/__mocks__/*.ts',
+    'src/platform/polyfills.ts',
```

Comment:
This appears to be a third party file: https://github.com/bluesky-social/social-app/blob/15c1b6ee157471807a723161066ba4ce5e12c0b5/src/platform/polyfills.ts#L6